.github/workflows/hardware-long.yml

```diff
@@ -11,17 +11,17 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  hardware-metal-test:
+  hardware-zephyr-test:
     strategy:
       fail-fast: false
       matrix:
         config:
           - board: p100
             runs-on:
               - yyz-zephyr-lab-p100
-          - board: p100a
-            runs-on:
-              - yyz-zephyr-lab-p100a
+          # - board: p100a
+          #   runs-on:
+          #     - yyz-zephyr-lab-p100a
```

Comment:
Just remove this - there is 4-year-old a bug in yamllint for it.

https://github.com/adrienverge/yamllint/issues/375